File: examples/nodejs/text-to-video.mjs
const response = await fetch('https://api.evolink.ai/v1/videos/generations', {
  method: 'POST',
  headers: {
    Authorization: `Bearer ${process.env.EVOLINK_API_KEY}`,
    'Content-Type': 'application/json'
  },
  body: JSON.stringify({
    model: 'seedance-2.0-fast-text-to-video',
    prompt: 'A moody neon-lit street at night, wet pavement, cinematic camera movement',
    duration: 5,
    quality: '720p',
    aspect_ratio: '16:9',
    generate_audio: true
  })
});

const data = await response.json();
console.log(data);

File: examples/python/text_to_video.py
import os
import requests

api_key = os.environ.get("EVOLINK_API_KEY")
if not api_key:
    raise SystemExit("Set EVOLINK_API_KEY first")

response = requests.post(
    'https://api.evolink.ai/v1/videos/generations',
    headers={
        'Authorization': f'Bearer {api_key}',
        'Content-Type': 'application/json'
    },
    json={
        'model': 'seedance-2.0-text-to-video',
        'prompt': 'A cinematic aerial shot over a futuristic city skyline at dawn',
        'duration': 5,
        'quality': '720p',
        'aspect_ratio': '16:9',
        'generate_audio': True,
    }
)

print(response.json())
